### DISCUSSION TITLE
audio db

### DISCUSSION THREAD

**Discussion by Aradhik11** [gokadzev/Musify#177] [General]
please how did you convert the auido into json format

**Comment by gokadzev** [comment]
> please how did you convert the auido into json format

Hello, what do you mean? there's no anything like that in the app currently.

**Comment by Aradhik11** [comment]
Hi, am trying to create a backend for my audio and integrate it in my
flutter app but in my backend I was unable to store the audio, so I now
want to ask did you know the process for me to achieve that?
